bin/trin/src/cli.rs

```diff
@@ -7,75 +7,39 @@ use clap::{
     error::{Error, ErrorKind},
     Args, Parser, Subcommand,
 };
-use url::Url;
-
-use crate::{
-    types::{bootnodes::Bootnodes, network::Subnetwork},
-    version::FULL_VERSION,
+use ethportal_api::{
```

Comment:
I feel like this refactor should have removed some dependencies from `ethportal-api`? At least `clap` ... maybe `url`?

Reply:
Good call!